src/visual.ts

```diff
@@ -256,7 +256,7 @@ module powerbi.extensibility.visual {
                 || _.isEmpty(categorical.Values[0].values)) {
                 return null;
             }
-
+            let counterValues: PrimitiveValue[] = dataView.metadata.columns.length > 2 && dataView.metadata.columns[2].roles['Counter'] ? dataView.categorical.values[1].values : null;
```

Comment:
Is `counter` always at second index? It'd be better to find index of the counter by its role name in order to be more robust